work16/index.js

```diff
@@ -0,0 +1,248 @@
+
+const REQUEST_URL = "https://mocki.io/v1/21ee587f-5a59-40ff-98c6-46b2d4715f25"
+
+// 間違っているURLの場合↓
+// const REQUEST_URL = "https://myjson.dit.upm.es/api/bins/ほげほげajy3";
+// 空の配列の場合↓
+// const REQUEST_URL = "https://mocki.io/v1/242b685f-a3d7-45a8-aeca-0376bd495b89";
+// 503エラーの場合↓
+// const REQUEST_URL = "https://httpstat.us/503";
+
+const request = async () => {
+    const response = await fetch(REQUEST_URL, {
+        headers: {
+            "Content-Type": "application/json"
+        }
+    });
+    if (!response.ok) {
+        console.log(`${response.status}:${response.statusText}`);
```

Comment:
IMO
I think it needs to be displayed in a way that users can understand. Currently, nothing is displayed on the screen, so I think users will be confused.

Reply:
I have modified the code. Is this code ok?

commit: [88f6478](https://github.com/yasu-pro/js-handson/pull/18/commits/88f6478340dc6c3486eee044359e403134a3529f)

Reply:
It is not appropriate to throw a new error here.
Ms.Morita told me that this is because it is difficult to tell whether the error is from the server or a reference error like a reference error.
I feel that it should be displayed on the spot.

Reply:
I thought I created the displayErrorMessage function for that purpose...
This area is not so easy to understand.

commit: [e5db9b3](https://github.com/yasu-pro/js-handson/pull/18/commits/e5db9b3f664d5de8029906fb69469bf4b3832a34)